src/pages/SignUp/SignUp.scss

```
@@ -0,0 +1,181 @@
+@mixin joinInput {
+  display: block;
+  width: 100%;
+  height: 51px;
+  border: solid 1px rgb(218, 218, 218);
+  background-color: rgb(255, 255, 255);
+  padding: 10px 110px 10px 14px;
+}
+@mixin smallFonts {
+  width: 100%;
+  height: 17px;
+  margin: 19px 0 8px;
+  font-size: 14px;
+  font-weight: 700;
+  text-align: left;
+}
+@mixin errorAlam {
+  display: block;
+  margin: 9px 0 -2px;
+  font-size: 12px;
+  line-height: 14px;
+  color: rgb(255, 0, 0);
+  text-align: left;
+}
+
+.signUp {
+  width: 100%;
+  height: 100vh;
+  background-color: rgb(245, 246, 247);
+  display: flex;
+  justify-content: center;
+  text-align: center;
+
+  .signUpWrap {
+    display: flex;
+    flex-direction: column;
+    // height: 100%;
+
+    .header {
+      margin: 0 15px;
+      padding: 59px 0 20px;
+
+      .logo {
+        text-decoration: none;
+        cursor: pointer;
+
+        .logoText {
+          font-size: 3em;
+          font-weight: bolder;
+          color: rgb(3, 199, 90);
+        }
+      }
+    }
+
+    .container {
+      width: 100%;
+      margin: 0 15px;
+      justify-content: center;
+
+      .rowGroup {
+        overflow: hidden;
+        width: 460px;
+        height: 351px;
+
+        .joinRow {
+          width: 100%;
+          height: 99px;
+
+          .joinTitle {
+            @include smallFonts;
+          }
+
+          .joinBox {
+            @include joinInput;
+
+            .typingArea {
+              width: 334px;
+              height: 29px;
+              padding-right: 29px;
+              border: none;
+            }
+          }
+
+          .errorNextBox {
+            @include errorAlam;
+          }
+        }
+
+        .joinRowPw {
+          width: 100%;
+          height: 215px;
+          .joinTitle {
+            @include smallFonts;
+          }
+
+          .joinBox {
+            @include joinInput;
+
+            .typingArea {
+              width: 334px;
+              height: 29px;
+              padding-right: 29px;
+              border: none;
+            }
+          }
+
+          .errorNextBox {
+            @include errorAlam;
+          }
+        }
```

```suggestion
        .joinRow, joinRowPw {
          width: 100%;

          .joinTitle {
            @include smallFonts;
          }

          .joinBox {
            @include joinInput;

            .typingArea {
              width: 334px;
              height: 29px;
              padding-right: 29px;
              border: none;
            }
          }

          .errorNextBox {
            @include errorAlam;
          }
        }
        
        .joinRow {
          height: 99px;
        }
        
        .joinRowPw {
          height: 215px;
        }
```
위와 같이 css 셀렉터를  , 를 사용해서 사용할 수도 있습니다!